README.md

```diff
@@ -85,9 +91,17 @@ WD-Firmware
 	- In the Import window select *General* -> *Existing Projects into Workspace* -> *Next*
 	- In the next Import window *Select root directory* -> *Browse*  and choose the infra-riscv-fw/WD-Firmware/ you've downloaded in 'Getting the firmware sources' section
 	- Press *'Finish'* button
-- From 'Eclipse MCU' menu bar select '*Project'* -> *'Build All'*
+- From 'Eclipse MCU' menu bar select '*Project'* -> *'Build All'*. Note that you can select which platform to build for.
+- You you will need to choose a specific ***demo*** for building a full sultion:
+
+        from the eclipse terminal or console:
+        $ cd [WD-firmware-root]/WD-Firmware/demo/build
+        $ python ./config.sh
+        
+        Then you will be asked to choose a demo.
+    For more explnation on adding new demos please read the readme file on ***'/demo'***

```

Comment:
In order to build there is also a need to follow one of the alternatives:
First option:
1) right click on WD-Firmware project caption
2) From the popping list, select: Build Configuration -> Set Active -> hifive1
3) Click Project -> Build all (or just Ctrl+B)

Second option:
From the console, at path /WD-Firmware/demo/build, type scons target=hifive1 + enter

Reply:
no need, this is Eclipse knowledge. we provide simple build.

Reply:
switched between bullets